CONTRIBUTING.md

```diff
@@ -25,13 +27,16 @@ Before being considered for review or merging, each pull request must:
 
 ## Building The Library
 
-The library should build and pass all tests.
+The library should build and pass all tests. The [protocol buffer compiler](https://github.com/protocolbuffers/protobuf) needs to be installed in order to generate protobufs.
```

Comment:
Any objection to adding this to save a click?
> For Mac users on brew, run `brew install protobuf`

Reply:
You could just take the commands from here: 
https://github.com/xpring-eng/xpring-common-protocol-buffers/blob/master/CONTRIBUTING.md#building-the-library